src/data/abilities.h

```diff
@@ -2273,6 +2273,7 @@ const struct Ability gAbilitiesInfo[ABILITIES_COUNT] =
         .cantBeSwapped = TRUE,
         .cantBeTraced = TRUE,
         .cantBeSuppressed = TRUE,
+        .cantBeOverwritten = TRUE,
```

Comment:
is there a source for that?

Reply:
It was added to upcoming on #5246 but not to master.